docs/compute-element/hosted-ce.md

```diff
@@ -0,0 +1,110 @@
+OSG Hosted CE
+=============
+
+
+## Overview
+
+As an alternative to setting up your own CE, OSG offers a hosted CE option
+to connect your existing campus HPC/HTC cluster to the Open Science Grid.
+The OSG team will host and operate the HTCondor compute element which routes user jobs to your cluster,
+configured for the science communities that you choose to support.
+
+![managed services diagram](/compute-element/img/screenshot_2983.png)
+
+
+## Requirements
+
+Here are the requirements:
+
+-   An existing compute cluster with a supported batch system (HTCondor, LSF, PBS, Slurm, SGE),
+    and a supported operating system (RHEL 6 or 7 or compatible)
+
+-   Outbound network connectivity from the compute nodes (they can be behind NAT)
+
+-   A Unix account on your cluster's submit server, accessible via an SSH key
+    The OSG CE will use this account to automatically submit jobs,
+    so it must also have permissions to submit jobs to the batch system
+
+-   If your batch system is not HTCondor,
+    there needs to be a shared file system between the submit server and the compute nodes
+
+
+## Setup Process
+
+Before making any system changes, you should do the following steps:
+
+-   Fill out the [cluster integration questionnaire](http://goo.gl/forms/8OukxsyG6KBSGHuR2)
+    so that the OSG team has basic information about your cluster
+
+-   Email [help@opensciencegrid.org](mailto:help@opensciencegrid.org)
+    to set up a consultation call with the OSG team,
+    in order to discuss how you would like to contribute to the OSG;
+    for example, the number of OSG jobs that should run, what resource limits you have,
+    or which science communities you support
+
+After the consultation, do the following:
+
+-   Create a Unix account on the submit server for use by the OSG CE
+
+-   Install public SSH keys for the account
+
+Once this is done, the OSG team will:
+
+-   Configure the OSG CE with your system details
+
+-   Validate operation with a set of test jobs
+
+-   Configure central OSG services to schedule jobs
+
+Optionally, we can assist you in installing and setting up the Squid and OASIS software on your cluster
+in order to support application software repositories.
+This will allow a broader set of jobs to run on your cluster.
+
+
+## Security
```

Comment:
Let's move the security section right below the requirements/before starting section